src/displayapp/screens/settings/SettingWatchFace.h

```diff
@@ -29,26 +32,16 @@ namespace Pinetime {
         auto CreateScreenList() const;
         std::unique_ptr<Screen> CreateScreen(unsigned int screenNum) const;
 
+        static constexpr int settingsPerScreen = 4;
+        std::array<Screens::CheckboxList::Item, UserWatchFaceTypes::Count> watchfaceItems;
+        static constexpr int nScreens = UserWatchFaceTypes::Count > 0 ? (UserWatchFaceTypes ::Count - 1) / settingsPerScreen + 1 : 1;
```

Comment:
Given that we assert that `UserWatchFaceTypes::Count >= 1`, we don't need the check here. Of course, it might be good to keep it just in case.

```suggestion
        static constexpr int nScreens = (UserWatchFaceTypes ::Count - 1) / settingsPerScreen + 1;
```

Reply:
Yes, I agree but... I have the feeling that if I read this code in a few months, I'll have forgetten about that asssert and this that there's a bug :-D. Since this is a `constexpr` evaluation, it shouldn't hurt to keep it. What do you think?

Reply:
There's definitely no harm in keeping it of course. I think it's fine as it is, I just noticed this as I was reading the code :-)